shoop_tests/campaigns/test_basket_contact_conditions.py

```diff
@@ -102,6 +102,16 @@ def test_group_basket_condition_with_anonymous_contact(rf):
 
 
 @pytest.mark.django_db
+def test_contact_group_basket_condition_with_none(rf):
+    request = rf.get("/")
+    request.shop = Shop()
+    basket = get_basket(request)
```

Comment:
No, I can't, because `basket.customer` will actually return AnonymousContact after this fix.